impit-python/python/impit/impit.pyi

```diff
@@ -0,0 +1,600 @@
+    """
+
+
+def put(
+    url: str,
+    content: bytes | bytearray | list[int] | None = None,
+    data: dict[str, str] | None = None,
+    headers: dict[str, str] | None = None,
+    timeout: float | None = None,
+    force_http3: bool | None = None,
+) -> ImpitPyResponse:
+    """Make a PUT request without creating a client instance.
+
+    Args:
+        url: URL to request
+        content: Raw content to send
+        data: unused
+        headers: HTTP headers
+        timeout: Request timeout in seconds
+        force_http3: Force HTTP/3 protocol
+
+    Returns:
+        Response object
+    """
+
+
+def patch(
+    url: str,
+    content: bytes | bytearray | list[int] | None = None,
+    data: dict[str, str] | None = None,
+    headers: dict[str, str] | None = None,
+    timeout: float | None = None,
+    force_http3: bool | None = None,
+) -> ImpitPyResponse:
+    """Make a PATCH request without creating a client instance.
+
+    Args:
+        url: URL to request
+        content: Raw content to send
+        data: unused
+        headers: HTTP headers
+        timeout: Request timeout in seconds
+        force_http3: Force HTTP/3 protocol
+
+    Returns:
+        Response object
+    """
+
+
+def delete(
+    url: str,
+    content: bytes | bytearray | list[int] | None = None,
+    data: dict[str, str] | None = None,
+    headers: dict[str, str] | None = None,
+    timeout: float | None = None,
+    force_http3: bool | None = None,
+) -> ImpitPyResponse:
+    """Make a DELETE request without creating a client instance.
+
+    Args:
+        url: URL to request
+        content: Raw content to send
+        data: unused
+        headers: HTTP headers
+        timeout: Request timeout in seconds
+        force_http3: Force HTTP/3 protocol
+
+    Returns:
+        Response object
+    """
+
+
+def head(
+    url: str,
+    content: bytes | bytearray | list[int] | None = None,
+    data: dict[str, str] | None = None,
+    headers: dict[str, str] | None = None,
+    timeout: float | None = None,
+    force_http3: bool | None = None,
+) -> ImpitPyResponse:
+    """Make a HEAD request without creating a client instance.
+
+    Args:
+        url: URL to request
+        content: Raw content to send
+        data: unused
+        headers: HTTP headers
+        timeout: Request timeout in seconds
+        force_http3: Force HTTP/3 protocol
+
+    Returns:
+        Response object
+    """
+
+
+def options(
+    url: str,
+    content: bytes | bytearray | list[int] | None = None,
+    data: dict[str, str] | None = None,
+    headers: dict[str, str] | None = None,
+    timeout: float | None = None,
+    force_http3: bool | None = None,
+) -> ImpitPyResponse:
+    ...
+
+
+def trace(
+    url: str,
+    content: bytes | bytearray | list[int] | None = None,
+    data: dict[str, str] | None = None,
+    headers: dict[str, str] | None = None,
+    timeout: float | None = None,
+    force_http3: bool | None = None,
+) -> ImpitPyResponse:
```

Comment:
The `options` and `trace` methods in clients have the docstrings - I would just copy-paste it here too (looks like omission).